src/groovy/au/org/emii/aatams/JodaOverrides.groovy

```diff
@@ -0,0 +1,42 @@
+package au.org.emii.aatams
+
+import org.joda.time.Interval
+import org.joda.time.base.AbstractInterval
+
+class JodaOverrides {
+
+    // Would be nicer to do this with an "Extension Module", but alas, that's
+    // only available in groovy 2.0.
+    //
+    // An extension module would avoid the need to mock/unmock in every test
+    // which executes the overlapsAny code.
+    static void apply() {
+        AbstractInterval.metaClass.contains = { List instantList ->
+            def owningInterval = delegate
+
+            null != instantList.find {
+                it && owningInterval.contains(it)
+            }
```

Comment:
We tend not to have the constant on the left hand side of the comparison. It should be:
``` groovy
instantList.find {
    it && owningInterval.contains(it)
} != null
```

If you think we should accept this style then I think we need to have an internal discussion about it. I know a few team members (in both camps) have strong feelings about it.